api/v1alpha1/openapi.yaml

```diff
@@ -189,6 +189,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/image/{id}/image-url:
```

Comment:
as per what we've discussed this will be under source:
maybe `api/v1/source/{id}/image-url`

Reply:
Right!